galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/FrameworkRuns.java

```diff
@@ -145,18 +146,17 @@ public FrameworkRuns(IFramework framework) throws FrameworkException {
 
     @Override
     @NotNull
-    public @NotNull IRun submitRun(String runType, String requestor, @NotNull String bundleName,
+    public @NotNull IRun submitRun(String runType, String requestor, String bundleName,
```

Comment:
probably need a better way of doing this.   I'm thinking on how we want to differentiate gherkin tests from java tests in Jenkins for example.

Reply:
We should have a chat about this on monday, see if its worth adding a competely new submit method